.pre-commit-config.yaml

```diff
@@ -0,0 +1,53 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.2.0
+    hooks:
+    -   id: trailing-whitespace
```

Comment:
Auto-removes trailing whitespace

Reply:
Note: pre-commit has a way to tell per-file hooks to skip file patterns/types. Skip patch files for this one. e.g., by exclude_types `diff` or by exclude  `\.patch$`